wunderio/next4drupal-project
# NEX-24: add example content, simplify frontent env var setup when using lando

This pr adds:

* simpler way to run the frontend in lando, by embedding all env vars in the lando environment and eliminating the need for the .env.local file
* added a command to stop the npm process when running in lando
* add migrations to populate the site with test content in all three languages. All content types and paragraphs are imported using test data added to csv files
* also includes josh's work on the frontend, merged to this branch in #48



### How to test: 

* this branch includes an updated version of the big command, run it and see that the pages now have content!

![image](https://user-images.githubusercontent.com/185412/220064632-e03b22f5-1029-40ec-b2c2-05e1c2fd3226.png)
